app/models/like.rb

```diff
@@ -1,2 +1,10 @@
 class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_like_counter_for_post
+    post = Post.find_by(id: post_id)
+    count = Like.where(post_id: post.id).count
+    post.update(LikesCounter: count)
```

Comment:
- [ ] There is an association between the post and the like, we can use direct posts & increment by 1
`post.increment!(:likes_counter)`

Reply:
Issue resolved, Thank you.